checkov/terraform/checks/resource/azure/FunctionAppMinTLSVersion.py

```diff
@@ -4,9 +4,15 @@
 
 class FunctionAppMinTLSVersion(BaseResourceValueCheck):
     def __init__(self):
+        """
+        The minimum supported TLS version for the function app.
+        Defaults to 1.2 for new function apps.
+        """
         name = "Ensure Function app is using the latest version of TLS encryption"
         id = "CKV_AZURE_145"
-        supported_resources = ['azurerm_function_app']
+        supported_resources = ['azurerm_function_app', 'azurerm_linux_function_app', 'azurerm_windows_function_app',
```

Comment:
it looks like the linux/windows variants changed the argument name a bit, it is `minimum_tls_version`. Can you adjust `get_inspected_key()` to differentiate between both variants, something like
```python
    def get_inspected_key(self) -> str:
        if self.entity_type in ("azurerm_function_app", "azurerm_linux_function_app"):
            return "site_config/[0]/min_tls_version"
        else:
            return "site_config/[0]/minimum_tls_version"
```
and also a test to validate it, thanks!

Reply:
line 2 in comment should be `if self.entity_type in ("azurerm_function_app", "azurerm_function_app_slot"):`.

Reply:
yeah, correct 😄